PlayCover/Views/Settings/IPASourceSettings.swift

```diff
@@ -42,7 +42,8 @@ struct IPASourceSettings: View {
         Form {
             HStack {
                 List(storeVM.sourcesList, id: \.id, selection: $selected) { source in
-                    SourceView(source: source)
+                    SourceView(source: source,
+                               isEnabled: storeVM.enabledList.contains(where: { $0 == source.source }))
```

Comment:
Now that I'm looking at this, it would be easier to have `StoreVM.enabledList` be a dictionary with the source name as the key and the value be if it is enabled or not. That would make the code cleaner.